src/frontends/onnx/frontend/src/frontend.cpp

```diff
@@ -135,6 +153,32 @@ bool FrontEnd::supported_impl(const std::vector<ov::Any>& variants) const {
 void FrontEnd::add_extension(const std::shared_ptr<ov::Extension>& extension) {
     if (auto telemetry = std::dynamic_pointer_cast<TelemetryExtension>(extension)) {
         m_extensions.telemetry = telemetry;
+    } else if (auto transformation = std::dynamic_pointer_cast<DecoderTransformationExtension>(extension)) {
+        m_transformation_extensions.push_back(transformation);
+    } else if (const auto& so_ext = std::dynamic_pointer_cast<ov::detail::SOExtension>(extension)) {
+        add_extension(so_ext->extension());
+        m_other_extensions.push_back(so_ext);
+    } else if (auto common_conv_ext = std::dynamic_pointer_cast<ov::frontend::ConversionExtension>(extension)) {
+        m_conversion_extensions.push_back(common_conv_ext);
+
+        for (int i = 1; i < 13; ++i)
+            ngraph::onnx_import::register_operator(common_conv_ext->get_op_type(),
+                                                   i,
+                                                   "",
+                                                   [=](const ov::frontend::onnx::Node& context) -> OutputVector {
+                                                       return common_conv_ext->get_converter()(NodeContext(context));
+                                                   });
+    } else if (const auto onnx_conv_ext = std::dynamic_pointer_cast<ConversionExtension>(extension)) {
+        m_conversion_extensions.push_back(onnx_conv_ext);
+
+        // todo: register in all opsets?
```

Comment:
@tomdol 
I didn't find a way how to register convertor for all onnx opsets, do we have such a method or we need to create a ticket to cover this?

Reply:
I've missed this comment yesterday. You should register the op just in opset 1. This will be used by models created using any opset `>=1`. It behaves like a fallback mechanism.

Reply:
The point is that the user may want to overwrite an existing operation, perhaps there is a bug or an unoptimized implementation, for any reason. In this case, the method you described will not work.

Reply:
@tomdol please double check this one, maybe we need to create a separate method to register/override a convertor for all opsets